### ISSUE TITLE
Duplicate decodes

### ISSUE THREAD

**Issue by mcogoni** [rtmrtmrtmrtm/weakmon#9]
Hi Robert,
I frequently observe duplicate decodes.
The problem is avoided by putting nchildren=1 in the process function.
You use the already_* dictionaries that work perfectly at the single process level, but they do not see each other in parallel operation. I know that you know this much better than me :)
I'd like to know if it is the intended behavior or you just didn't bother to do a final dupe check after all processes end. It's not a big deal since I can always filter out the dupes in post processing, but it would be nice to have a perfect output already from ft8.py

marco
